OpenRA.Game/Graphics/SpriteRenderer.cs

```diff
@@ -51,7 +51,7 @@ public void Flush()
 				shader.PrepareRender();
 
 				// PERF: The renderer may choose to replace vertices with a different temporary buffer.
-				renderer.DrawBatch(ref vertices, nv, PrimitiveType.TriangleList);
+				renderer.DrawBatch(shader, vertices, nv, PrimitiveType.TriangleList);
```

Comment:
Accidentally dropped the `ref` which I expect will break things.

Reply:
Fixed